<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecognizeDialog</class>
 <widget class="QDialog" name="RecognizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="cmb_select_option">
     <item>
      <property name="text">
       <string>Train</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Recognize</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stk_train_recognize">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_train">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gpb_train">
         <property name="title">
          <string>Train</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QWidget" name="wdg_local_cloud" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="rdb_train_local">
               <property name="text">
                <string>Local</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rdb_train_cloud">
               <property name="text">
                <string>Cloud</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="stk_local_cloud">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_local">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="lbl_input_path_local_label">
                <property name="text">
                 <string>Input path:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lie_input_path_local_value"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_cloud">
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="lyt_h_cloud_user">
                <item>
                 <widget class="QLabel" name="lbl_cloud_user_label">
                  <property name="text">
                   <string>User:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lie_cloud_user_value"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="lyt_h_cloud_password">
                <item>
                 <widget class="QLabel" name="lbl_cloud_password_label">
                  <property name="text">
                   <string>Password:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lie_cloud_password_value"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="lyt_h_cloud_token">
                <item>
                 <widget class="QLabel" name="lbl_cloud_token_label">
                  <property name="text">
                   <string>Token:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lie_cloud_token_value"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_go_train">
            <property name="text">
             <string>GO!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_recognize">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="gpb_recognize">
         <property name="title">
          <string>Recognize</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lbl_trained_file_label">
            <property name="text">
             <string>Trained file:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lie_trained_file_value"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_go_recognize">
            <property name="text">
             <string>GO!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
